packages/manager/apps/dedicated/client/app/account/user/user.routes.js

```diff
@@ -20,6 +20,10 @@ angular.module('UserAccount').config(
       },
       redirectTo: `${name}.method`,
       resolve: {
+        angularQr: /* @ngInject */ ($ocLazyLoad) =>
+          import('angular-qr').then((module) =>
```

Comment:
👍 can this be imported in a lazy loaded module?

Reply:
yes importing it in a lazy loaded module would have the same effect (or even better since the whole module would only be loaded when needed)
i didn't look at lazy load user module because it would take much more refactoring work, it's just a quick win to reduce the bundle size